lib/maths/unittest/CBoostedTreeTest.cc

```diff
@@ -246,7 +246,7 @@ void CBoostedTreeTest::testPiecewiseConstant() {
             0.0, modelBias[i][0],
             4.0 * std::sqrt(noiseVariance / static_cast<double>(trainRows)));
         // Good R^2...
-        CPPUNIT_ASSERT(modelRSquared[i][0] > 0.96);
+        CPPUNIT_ASSERT(modelRSquared[i][0] > 0.95);
```

Comment:
Good question! This is because I pass the random number generator by value to the function which generates the cross validation row masks. Before, we were using it directly and so we get different random numbers generated after this point. This happens very early during training, so affects most of random numbers generated.